components/sp-solutions/org.wso2.carbon.sp.solutions.apim.analytics/pom.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>sp-solutions</artifactId>
+        <groupId>org.wso2.analytics.solutions</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+    <name>WSO2 Stream Processor - Apim analytics solution</name>
+    <artifactId>org.wso2.carbon.sp.solutions.apim.analytics</artifactId>
+
+    <modules>
+        <module>components/org.wso2.sp.extension.siddhi.io.mgwfile</module>
+        <module>components/org.wso2.analytics.apim.rest.api.file</module>
+        <module>widgets</module>
+    </modules>
+
+</project>
```

Comment:
add new line EOF

Reply:
Fixed in a21aa920ceb5fdbd3f34fa9dddb75b5630876048 of https://github.com/wso2/analytics-apim.